src/components/ArtworkItem.jsx

```diff
@@ -21,6 +21,20 @@ const ArtworkItem = ({ data }) => (
           {data.creation_date}
         </p>
       </div>
+      {
+        // Para el testo
```

Comment:
Sorry, can you explain here?

Reply:
A comment that escaped me.
I wanted to leave a section to test the requests